data-pipeline/tests/unit/pre_processing/conftest.py

```diff
@@ -309,6 +311,7 @@ def aar_central_services_data() -> pd.DataFrame:
             "BNCH21606 (Agency supply teaching staff)": [1000, 1001, 1002],
             "BNCH21403 (Energy)": [1000, 1001, 1002],
             "BNCH21402 (Water and sewerage)": [1000, 1001, 1002],
+            "BNCHBAI061 (Coronavirus Govt Funding)": [1000, 1001, 1002],
```

Comment:
I'm stumped by this one: it wasn't in this `dict` but the tests pass on `main`…?!

Reply:
So it's not that they didn't actually run?!

Reply:
I've checked out `main` and run this test and it passes: this column _isn't_ there.

…but it _is_ there now. Need to account for where it was being dropped.

Reply:
This is indeed down to a change I made:

```python
central_services_financial = pd.read_csv(
    cs_path,
    encoding="utf-8",
    usecols=lambda x: x in input_schemas.aar_central_services.keys(),
    dtype=input_schemas.aar_central_services,
)
```

…versus:

```python
central_services_financial = pd.read_csv(
    cs_path,
    encoding="utf-8",
    usecols=input_schemas.aar_central_services.keys(),
    dtype=input_schemas.aar_central_services,
)
```

The former will check whether any columns in the input file are in the expected schema and, if so, include them.

I'd argue the latter is correct, otherwise we're processing inconsistently-structured data but for the sake of confusion, I'll revert.
The latter will include columns from the expected schema and, if they're _not_ they're, still include the column.

Reply:
Reverted.